artifacts/camara-cloudevents/event-subscription-template.feature

```diff
@@ -0,0 +1,366 @@
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_32_get_unknown_<xxx>_subscription_for_a_device
+  Scenario:  Get method for <xxx> subscription with subscription-id unknown to the system  
+    Given the path parameter "subscriptionId" is set to a value not corresponding to any existing subscription
+    When the request "retrieve<xxx>Subscription" is sent
+    Then the response  code is 404
+    And the response property "$.status" is 404
+    And the response property "$.code" is "NOT_FOUND"
+    And the response property "$.message" contains a user friendly text
+
+########################### Subscription list retrieval scenarios #####################################
+
+  @<xxx>_subscriptions_60_no_authorization_header_for_list_subscription
+  Scenario: No Authorization header for list subscription
+    Given header "Authorization" is not present
+    When the request "retrieve<xxx>SubscriptionList" is sent 
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_61_expired_access_token_for_list_subscription
+  Scenario: Expired access token for list subscription
+    Given the header "Authorization" is set to expired token
+    When the request "retrieve<xxx>SubscriptionList" is sent
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+    
+  @<xxx>_subscriptions_62_invalid_access_token_for_list_subscription
+  Scenario: Invalid access token for list subscription
+    Given the header "Authorization" set to an invalid access token
+    When the request "retrieve<xxx>SubscriptionList" is sent
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+########################### Subscription deletion scenarios #####################################
+
+  @<xxx>_subscriptions_33_no_authorization_header_for_delete_subscription
+  Scenario: No Authorization header for delete subscription
+    Given header "Authorization" is set without a token
+    When the request "delete<xxx>Subscription" is sent 
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_34_expired_access_token_for_delete_subscription
+  Scenario: Expired access token for delete subscription
+    Given header "Authorization" is set with an expired token
+    When the request "delete<xxx>Subscription" is sent

```

Comment:
Missing step: And path parameter "subscriptionId" is set to the identifier of an existing subscription